build/sections/founder.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+    <div class="pt-36">
+        <div class="flex flex-col w-full justify-center md:justify-start text-white font-raleway md:w-[80%] py-5">
```

Comment:
By default I belive `div` has `w-full`. So,check it well again if you're not being redundant here.